include/bredis.hpp

```diff
@@ -0,0 +1,19 @@
+//
+//
+// Copyright (c) 2017 Ivan Baidakou (basiliscos) (the dot dmol at gmail dot com)
```

Comment:
It's 2019 now, as far as I know :) And you are `Stephen Coleman`, right? :)

Reply:
lol right, I just copied from another random file. shall I change it?

Reply:
yep, sure u are the author :)